src/Microsoft.AspNetCore.Razor.VSCode/src/RazorLanguageServerClient.ts

```diff
@@ -73,6 +75,10 @@ export class RazorLanguageServerClient implements vscode.Disposable {
         this.eventBus = new EventEmitter();
     }
 
+    public onStarted(listener: () => Promise<any>) {
+        this.onStartedListeners.push(listener);
```

Comment:
Is there any scenario where someone could try to register an `onStarted` listener after it's already started? If so, what's the desired behavior?

 * We could call their listener callback immediately
 * We could ignore them

The code as-is would accumulate extra listeners in `onStartedListeners` but never do anything with them. This is fine if we have reason to believe the scenario will never occur.

Reply:
Scenario will never occurr